core/imgread/common.cpp

```diff
@@ -188,16 +188,7 @@ bool InitDrive(u32 fileflags)
 #else
 	int gfrv=0;
```

Comment:
I would also init here to some symbolic `rv_*` value just to make the code nicer